### PR TITLE
Example/facebook pixel script

### PR DESCRIPTION
**Pull request by lukeshumard**
The [with-facebook-pixel](https://github.com/vercel/next.js/tree/canary/examples/with-facebook-pixel) example attempts to run the <Script /> loading the Facebook Pixel on every request. This results in a warning stating `[Meta Pixel] - Duplicate Pixel ID: xxx`. According to the Next.js docs, by using an [id on the inline script](https://nextjs.org/docs/basic-features/script#inline-scripts) Next is able to track and optimize the script. Since this is absent on the present example, the inline script is continuously loaded on every request.

I've added an id to this inline script, and the warning from facebook is no longer present.

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [x] Make sure the linting passes by running `yarn lint`

### CODE REVIEW

**Review by Auta1994 — COMMENTED** [review]
Sani

File: examples/with-facebook-pixel/package.json [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
+    "lint": "next lint",
     "start": "next start"
   },
   "dependencies": {
     "next": "latest",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "eslint": "8.9.0"
```
Comment (balazsorban44) [inline examples/with-facebook-pixel/package.json:15]:
```suggestion
    "start": "next start"
  },
  "dependencies": {
    "next": "latest",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
  "devDependencies": {
```

I don't think this was a necessary change, could we revert it?

Comment (balazsorban44) [inline examples/with-facebook-pixel/package.json:15]:
```suggestion
    "start": "next start"
  },
  "dependencies": {
    "next": "latest",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
```


I don't think this was a necessary change, could we revert it?

Comment (lukeshumard) [reply examples/with-facebook-pixel/package.json:15]: I can revert this if needed. When reading the template for contributing, I saw...
> Make sure the linting passes by running `yarn lint`
...which is why I added in the necessary packages for linting. Without those packages, `yarn lint` does not work. Let me know if you'd like me to revert adding the linting.

Comment (balazsorban44) [reply examples/with-facebook-pixel/package.json:15]: The Next.js repo has a linter already. :+1:

It's referring to that: https://github.com/vercel/next.js/blob/c7b2083f2eff850150ae6ac315d04ac74737e880/package.json#L26

Comment (lukeshumard) [reply examples/with-facebook-pixel/package.json:15]: Ah, got it! Sorry for the confusion. I'll revert those changes then.

File: examples/with-facebook-pixel/package.json [hunk 2]
```diff
@@ -10,4 +10,5 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
   }
+  }
```
Comment (balazsorban44) [inline examples/with-facebook-pixel/package.json:13]:
```suggestion
```

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**